PacketDotNet/TcpFields.cs

```diff
@@ -85,7 +89,7 @@ static TcpFields()
             SequenceNumberPosition = DestinationPortPosition + PortLength;
             AckNumberPosition = SequenceNumberPosition + SequenceNumberLength;
             DataOffsetPosition = AckNumberPosition + AckNumberLength;
-            FlagsPosition = DataOffsetPosition + DataOffsetLength;
+            FlagsPosition = DataOffsetPosition;// + DataOffsetLength;
```

Comment:
I noticed your other commit removed the stray comment. You can rebase this commit and remove the introduction of the comment before you ever add it.